contents/blog/posthog-alternatives.md

```diff
@@ -0,0 +1,726 @@
+        	<td className="w-3/12 text-center"><strong>Amplitude</strong></td>
+        	<td className="w-3/12 text-center"><strong>PostHog</strong></td>
+        	<td className="w-3/12 text-center"></td>
+    	</tr>
+	</thead>
+	<tbody>
+		<tr>
+        	<td>Product analytics</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Analyze user behavior</td>
+    	</tr>
+		<tr>
+        	<td>Autocapture</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Fast to implement</td>
+    	</tr>
+		<tr>
+        	<td>Session replay</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Watch real user sessions</td>
+    	</tr>
+    	<tr>
+        	<td>Feature flags</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+          <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Manage feature access and rollouts</td>
+    	</tr>
+    	<tr>
+        	<td>A/B testing</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">CRO and test in production</td>
+    	</tr>
+    	<tr>
+        	<td>Group analytics</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Analyze groups (e.g. companies and teams)</td>
+    	</tr>
+    	<tr>
+        	<td>SQL access</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Write your own queries and filters</td>
+    	</tr>
+    	<tr>
+       	<td>Transparent pricing</td>
+      		<td className="text-center"><span className="text-red text-lg">✖</span></td>
+       	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Know exactly what you'll pay</td>
+    	</tr>
+    	<tr>
+       	<td>Billing limits</td>
+      		<td className="text-center"><span className="text-red text-lg">✖</span></td>
+       	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Prevent surprise bills</td>
+    	</tr>
```

Comment:
this too sound the same? Dont think we need both. I'd remove "prevent surprise bills"